apps/send/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "send",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "devDependencies": {
+    "@types/node": "20.5.9",
+    "@types/react": "18.2.21",
+    "@types/react-dom": "18.2.7",
+    "autoprefixer": "10.4.15",
+    "eslint": "8.48.0",
+    "eslint-config-next": "13.4.19",
+    "next": "13.4.19",
+    "postcss": "8.4.29",
```

Comment:
For the goal of this activity, do we need this dependency?

Reply:
eslint is necessary for checking potential errors while compiling, better to have it even if is a basic starting installation. Postcss was installed from next.js directly, I'll remove it for now

Reply:
I pointed the comment to `postcss` and I'd remove it from this PR. I agree to keep `eslint` 😄